src/zend/try_catch.rs

```diff
@@ -26,14 +28,25 @@ pub(crate) unsafe extern "C" fn panic_wrapper<R, F: FnMut() -> R + RefUnwindSafe
 ///
 /// * `Ok(R)` - The result of the function
 /// * `Err(CatchError)` - A bailout occurred during the execution
-pub fn try_catch<R, F: FnMut() -> R + RefUnwindSafe>(func: F) -> Result<R, CatchError> {
+pub fn try_catch<R, F: FnMut() -> R + RefUnwindSafe>(
+    func: F,
+    first: bool,
```

Comment:
not sure about adding a parameter here, instead of having another function which should be more explicit

Reply:
IMHO, I would create a new function that reflect the underlying zend api